views/campaign-form-add.twig

```diff
@@ -25,14 +25,23 @@
 {% block formHtml %}
     <div class="row">
         <div class="col-md-12">
-            <form id="campaignAddForm" class="XiboForm form-horizontal" method="post" action="{{ urlFor("campaign.add") }}">
+            <form id="campaignAddForm" class="XiboForm form-horizontal" method="post" action="{{ urlFor("campaign.add") }}" data-gettag="{{ data.gettag }}">
```

Comment:
it would be better to use urlFor directly here, rather than passing it from the controller